noxtest/test_platform.go

```diff
@@ -0,0 +1,32 @@
+package noxtest
+
+import (
+	"time"
+)
+
+// Platform implementation mock to allow test using ticks
```

Comment:
```suggestion
// MockPlatform is a mock Platform implementation to allow test using ticks
```
(godoc should start with an exact name)